docs/source/getting_started_guide/installing_receptor.rst

```diff
@@ -0,0 +1,14 @@
++++++++++++++++++++
```

Comment:
Can we not reuse the existing installation guide found in `docs/source/installation.rst` ?

Reply:
The idea behind a getting started is to make it quick and easy. So in many cases, no, we don't want to reuse an existing install guide.

In this instance, you could go either way since the actual install guide is not much more complex than this pip install section.

Reply:
Cool, thanks for the feedback @samccann 

@thom-at-redhat  for this install guide, if we are not linking to the existing guide then the content of this page is not complete. This page only describes installing receptorctl, which will not work without the receptor binary installed.